package.json

```diff
@@ -85,15 +86,13 @@
     "eslint-config-stylelint": "^7.0.0",
     "file-exists-promise": "^1.0.2",
     "flow-bin": "^0.54.0",
-    "fs-promise": "^2.0.3",
```

Comment:
I know this package has been deprecated in favour of `fs-extra` per https://www.npmjs.com/package/fs-promise but the `pify` package has not been added via npm install pify --save-dev` to replace `fs-promise` here in `package.json`

Reply:
I believe `pify` is already available [as a dependency](https://github.com/stylelint/stylelint/blob/90149a5e650e1adb138206b29e055847da59f4ba/package.json#L60)

Reply:
That it is, my bad, sorry 😏